src/platform/site-wide/wc-loader.js

```diff
@@ -3,7 +3,7 @@ import '@department-of-veterans-affairs/component-library/dist/main.css';
 import {
   applyPolyfills,
   defineCustomElements,
-} from '@department-of-veterans-affairs/component-library';
+} from '@department-of-veterans-affairs/web-components/loader';
```

Comment:
The reexported "core" build is done with webpack 5, and we've noticed issues with it working on IE11. This avoids that and uses the build from the `web-components` child package which we know works on IE11.